### ISSUE TITLE
signing_key attribute works during --init but not during tflint

### ISSUE THREAD

**Issue by iwarapter** [terraform-linters/tflint#1165]
```hcl
plugin "pingaccess" {
    enabled = true
    version = "0.0.1-alpha"
    source  = "github.com/iwarapter/tflint-ruleset-pingaccess"

    signing_key = <<-KEY
    -----BEGIN PGP PUBLIC KEY BLOCK-----

    mQINB...
    -----END PGP PUBLIC KEY BLOCK-----
    KEY
}

```

## Version

```console
$ tflint --version
TFLint version 0.30.0
$ tflint --init                                  
Installing `pingaccess` plugin...
Installed `pingaccess` (source: github.com/iwarapter/tflint-ruleset-pingaccess, version: 0.0.1-alpha)
$ tflint                                         
Failed to apply config to plugins. An error occurred:

Error: .tflint.hcl:59,8-8: Unterminated template string; No closing marker was found for the string.

```

line 59 is the final `KEY` marker for the heredoc on the signing_key attribute. Once installed I can remove the `signing_key` block and the plugin has been installed and works correctly, it just doesn't seem to be able to parse the `signing_key` heredoc. 

This ruleset is currently private whilst I finish testing before OSS'ing it, and this is the last test :joy:
